pkg/indexmanagement/scripts.go

```diff
@@ -34,8 +34,16 @@ CMD=$(cat <<END
 from __future__ import print_function
 import json,sys
 r=json.load(sys.stdin)
+if not isinstance(r,dict):
+  print("Unable to determine write indices. Response was not a json payload")
+  print(r)
+  sys.exit()
 alias="${POLICY_MAPPING}-write"
-indices = [index for index in r if r[index]['aliases'][alias]['is_write_index']]
+indices = []
+for index in r:
+  if isinstance(r[index],dict):
+    if 'aliases' in r[index] and alias in r[index]['aliases'] and 'is_write_index' in r[index]['aliases'][alias]:
```

Comment:
IIRC we should also be doing null checks for cases where (as an example) `r[index]['aliases'][alias]` is empty